src/scrot.h

```diff
@@ -40,4 +40,5 @@ Window scrotGetWindow(Display *, Window, int, int);
 int scrotGetGeometry(Window, int *, int *, int *, int *);
 void scrotNiceClip(int *, int *, int *, int *);
 void scrotDoDelay(void);
+void scrotWaitFor(int);
```

Comment:
`scrotSleepFor` might've been a better name, but I tried to stay consistent with `scrotWaitUntil`.

Reply:
You can change `scrotWaitUntil` to `scrotSleepUntil`, I'm indifferent to the name.